docs/deploy/providers/heroku.md

```diff
@@ -23,3 +23,47 @@ Nitro supports deploying on [Heroku](https://heroku.com/) with minimal configura
    ```bash
    heroku config:set NITRO_PRESET=heroku
    ```
+   
+1. Configure your app.
+
+   ```bash
+   heroku config:set NITRO_PRESET=heroku
+   ```
+   
+1. Configure your app.
+
+   ```bash
+   heroku config:set HOST=0.0.0.0
+   ```
+
+## Update your config
+
+In order to run Nuxt 3 on Heroku, you need to update the following packages.
+
+1. Add to `nuxt.config.ts` file the `nitro` preset.
+
+   ```
+   import { defineNuxtConfig } from 'nuxt';
+
+   // https://v3.nuxtjs.org/api/configuration/nuxt.config
+   export default defineNuxtConfig({
+     nitro: {
+       preset: 'node-server',
+     },
+   });
+   ```
+1. Add `start` and `heroku-postbuild` commands to `package.json` file.
+
+   ```
+   "scripts": {
+      "heroku-postbuild": "npm run build",
+      "start": "nuxt preview"
```

Comment:
```suggestion
      "start": "node .output/server/index.mjs"
```

Reply:
How was it working before without start @danielroe ?

Reply:
Likely because the starter package had `"start": "node .output/server/index.mjs"` already present.